tests/unit/helpers.js

```diff
@@ -182,10 +182,17 @@ export function fake<T>(original: Object, methods: Object = {}): T {
  * Returns a fake Firefox client as would be returned by
  * connect() of 'node-firefox-connect'
  */
+
+type FakeFirefoxClientParams = {
```

Comment:
Please use pipes for this and all type definitions. This will make sure that no *unknown* object keys are allowed. Example:
````javascript
type FakeFirefoxClientParams = {|
  requestResult?: Object,
|}
````